packages/block-library/src/media-text/index.js

```diff
@@ -66,6 +66,9 @@ const blockAttributes = {
 		type: 'boolean',
 		default: false,
 	},
+	verticalAlignment: {
+		type: 'string',
```

Comment:
Would it make sense to have a default value of center?

Reply:
We established a convention in the [parent PR](https://github.com/WordPress/gutenberg/pull/13899) (Columns) that defaults would not be applied as this could be confusing for existing Themes. So default to `null` ("leave as is") so as to retain backwards compat.